packages/stats/gbstats/gbstats.py

```diff
@@ -480,19 +484,19 @@ def format_variation_result(
             )
         else:
             power_response = None
-        frequentist = row[f"{prefix}_p_value"] is not None
         testResult = {
             "expected": row[f"{prefix}_expected"],
             "uplift": row[f"{prefix}_uplift"],
             "ci": row[f"{prefix}_ci"],
             "errorMessage": row[f"{prefix}_error_message"],
         }
-        if frequentist:
+        if row["engine"] == "frequentist":
```

Comment:
I also don't like relying on pandas to pass this stuff around, but that's for a later refactor. No need to change anything here, this is good for now.